buildkite/scripts/cache-through.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -eou pipefail
+set +x
+
+if [[ $# -ne 2 ]]; then
+  echo "Usage: $0 <path-to-file> <postprocess-run-preprocess-in-docker-cmd>"
+  exit 1
+fi
+
+# download gsutil if it doesn't exist
+# TODO: Bake this into the agents
+if [[ ! -f ./google-cloud-sdk/bin/gsutil ]]; then
+  echo "Downloading gsutil because it doesn't exist"
+  wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-296.0.1-linux-x86_64.tar.gz && tar -zxf google-cloud-sdk-296.0.1-linux-x86_64.tar.gz
```

Comment:
may be easier to reason about `tar`'ing into a specific, more global DIR like */usr/local/* or */opt/*. with `-C/--directory`.

Reply:
good idea